dashboard/test/test_helper.rb

```diff
@@ -494,9 +494,9 @@ def storage_decrypt_channel_id(encrypted)
   return [storage_id, channel_id]
 rescue
   raise ArgumentError if encrypted.nil?
-  _storage_id, channel_id = encrypted.split('-')[1, 2]
+  storage_id, channel_id = encrypted.split('-')[1, 2]
   raise ArgumentError if channel_id.nil?
-  return channel_id.to_i
```

Comment:
Glad to see the return value signature has been fixed to match what the non-stubbed method returns.

Reply:
Now that you mention it, I should probably also remove the sentence of the comment that starts on line 480

Reply:
perhaps that's grounds to downgrade from "worse than atrocious" to just "atrocious"? :trollface: